src/main/java/org/codehaus/mojo/versions/DisplayDependencyUpdatesMojo.java

```diff
@@ -91,6 +86,50 @@
     @Parameter( property = "processDependencyManagementTransitive", defaultValue = "true" )
     private boolean processDependencyManagementTransitive;
 
+    /**
+     * Only take these artifacts into consideration.
+     *
+     * Comma- and/or whitespace-separated list of extended GAV patterns.
+     *
+     * <p>
+     * Extended GAV: groupId:artifactId:version:type:classifier:scope
+     * </p>
+     * <p>
+     * The wildcard "*" can be used as the only, first, last or both characters in each token.
+     * The version token does support version ranges.
+     * </p>
+     *
+     * <p>
+     * Example: "mygroup:artifact:*, *:*:*:*:*:compile"
+     * </p>
+     *
+     * @since 2.11
+     */
+    @Parameter(property = "dependencyManagementIncludes", defaultValue = WildcardMatcher.WILDCARD)
+    private String dependencyManagementIncludes;
```

Comment:
Why not use List<String> ... 
will be clear in configuration and in code you not need parse it.

https://maven.apache.org/guides/mini/guide-configuring-plugins.html#mapping-lists

Reply:
I actually  had this implementation once :)
But: I did not find any combination of escapes to make maven-invoker-plugin pass this correctly to the versions plugin.

AbstractVersionsDependencyUpdaterMojo#separatePatterns seems to also have some kind of parser for this.
In combination with [this post](https://stackoverflow.com/questions/1335291/string-arrays-as-command-line-arguments-for-maven-plugin) I just assumed, I had to implement it this way.

Do you know a project/mojo where I could get inspiration from?

Reply:
For invoker you can try put properties in file `test.properties`

With newer Maven should work mapping property to collections - separator is comma

> Do you know a project/mojo where I could get inspiration from?

Good question .... you can read https://maven.apache.org/plugin-developers/index.html or ask on Maven dev list.

By the way thanks for work, looks good - for first PR

Reply:
Thanks for your hints!

Seems like the source for my original problem is some weird combination of properties parsing ([probably this](https://stackoverflow.com/questions/21028164/value-in-properties-file-split-by-comma-reading-file-only-loads-first-element)) and flattening in the runner plugin.

Switching to List\<String> works if using test.properties as you suggested, see latest commits.